Makefile

```diff
@@ -148,3 +148,11 @@ swagger-gen:
 		--entrypoint hack/generate-swagger-api.sh \
 		-e GOPATH=/go \
 		quay.io/goswagger/swagger:0.7.4
+
+.PHONY: swagger-docs
+swagger-docs:
```

Comment:
can you add a `## ` comment here? We use it for `make help` to show a nice usage description :heart: